fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchTest.java

```diff
@@ -639,6 +639,9 @@ public void testSearchObservationWithPatientCompartment() {
         Bundle bundle = response.readEntity(Bundle.class);
         assertNotNull(bundle);
         assertTrue(bundle.getEntry().size() >= 1);
+        // verify link does not include consecutive '&' characters
+        assertTrue(bundle.getLink().size() >= 1);
+        assertTrue(!bundle.getLink().get(0).getUrl().getValue().contains("&&"));
```

Comment:
For Your Awareness... assertFalse is a good choice here.

Reply:
Updated.